canonical-websites/snapcraft.io
# Convert screenshots to React and implement new design

Fixes https://github.com/canonical-websites/snapcraft.io/issues/1671

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/<snap_name>/listing
- You should be able to manipulate screenshots successfully with a mouse
- You should be able to manipulate screenshots successfully with a keyboard (tab, delete, return keys)
- If you have over 5 images uploaded (blender) you should receive a warning with slightly different styled images
- Turn JS off and you should still get a read-only view of your screenshots
